src/server/ua_subscription_events.c

```diff
@@ -0,0 +1,562 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2018 (c) Ari Breitkreuz, fortiss GmbH
+ */
+
+#include "ua_server_internal.h"
+#include "ua_subscription_events.h"
+
+#ifdef UA_ENABLE_EVENTS
+
+typedef struct node_iteration {
+    LIST_ENTRY(node_iteration) listEntry;
+    UA_NodeId *node;
+} node_iteration;
```

Comment:
use a more unique name. This generic name could collide with other definitions in stack/user code, if amalgamation is used. Same for other global definitions within this file.